docs/quickstart.md

```diff
@@ -88,14 +88,9 @@ you can follow the rest of the guide the same way as if not using docker.
 
 ## Preliminaries
 
-We have created a  <a href="https://github.com/fetchai/agents-template" target="_blank">template repo for AEA development</a> here which you can optionally fork and use immediately.
+Follow `Approach 1` in the <a href="../development-setup#approach-1">development setup</a> guide.
 
-Alternatively, create and enter into a new working directory:
-
-``` bash
-mkdir my_aea_projects/
-cd my_aea_projects/
-```
```

Comment:
Only if they use template repo do they have a clean working dir already, so not sure why this is removed

Reply:
I directed the user to follow the development-guide page (which includes both manual and template-repo options). 
https://docs.fetch.ai/aea/development-setup/#approach-1

They will have a working directory in either case, so the directory creation instruction can be removed.